astroquery/utils/tap/core.py

```diff
@@ -791,12 +792,19 @@ def __init__(self, url=None,
                                       connhandler=connhandler,
                                       verbose=verbose)
         self.__internalInit()
+        self.__set_client_id(client_id=client_id)
 
     def __internalInit(self):
         self.__user = None
         self.__pwd = None
         self.__isLoggedIn = False
 
+    @staticmethod
+    def __set_client_id(client_id):
+        if client_id:
+            global TAP_CLIENT_ID

```

Comment:
According to [Python naming conventions](https://peps.python.org/pep-0008/#constants) `TAP_CLIENT_ID` is supposed to be (and so far has been) a module level constant. Now it should be changed to be an instance attribute instead.

Reply:
Modified TAP_CLIENT_ID to be an instance attribute.